src/benchmarks/micro/libraries/Microsoft.Extensions.Logging/LoggingBenchmarkBase.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Logging.Performance
+{
+    public class LoggingBenchmarkBase
+    {
+        protected static readonly Action<ILogger, Exception> NoArgumentTraceMessage = LoggerMessage.Define(LogLevel.Trace, 0, "Message");
+        protected static readonly Action<ILogger, Exception> NoArgumentErrorMessage = LoggerMessage.Define(LogLevel.Error, 0, "Message");
+
+        protected static readonly Action<ILogger, int, string, Exception> TwoArgumentTraceMessage = LoggerMessage.Define<int, string>(LogLevel.Trace, 0, "Message {Argument1} {Argument2}");
+        protected static readonly Action<ILogger, int, string, Exception> TwoArgumentErrorMessage = LoggerMessage.Define<int, string>(LogLevel.Error, 0, "Message {Argument1} {Argument2}");
+
+        protected static Exception Exception = ((Func<Exception>)(() => {
+            try
+            {
+                throw new Exception();
+            }
+            catch (Exception ex)
+            {
+                return ex;
+            }
+        }))();
```

Comment:
nit: it took me a second or two to understand what this code is doing. We could use an old school method here ;)

```suggestion
        protected static Exception Exception = GetRealException();
        
        private static Exception GetRealException()
        {
            try
            {
                throw new Exception();
            }
            catch (Exception ex)
            {
                return ex;
            }
        }
```